.github/workflows/preprod-ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Preproduction Release
```

Comment:
The concept here is that we would merge to `pre-prod` as our initial merge of any changes, and downstream, teams could use the `@pre-prod` tag in dev/qa to only test merged changes in those environments.

We could still use our canary release for ad-hoc changes, although those may be unnecessary if we create a more robust system of creating dist tags for PRs.

Reply:
@aabenoja to follow up on your line of questioning, I've started modeling this behavior in the homepage.

Here's the [Octopus variable defining the version](https://octopus.mktp.io/app#/Spaces-1/projects/homepage-micro-front-end/variables?name=es-components-environment).

Here's the [webpack.config utilizing that value](https://github.com/im-web-experience/homepage/blob/es-components-update/src/Homepage.Bff/FrontEnd/webpack.config.js#L16).